apps/etl-func/src/domain/interfaces/tokenizer.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+export interface TokenizerClient {
+  tokenize(pii: string): Promise<string>;
+}
+
+const messageTokenizedSchema = z.object({
+  fiscalCode: z.string().uuid().min(1),
```

Comment:
is `uuid()` correct here?

Reply:
The token that the tokenizer api returns is a uuid